PWGHF/vertexingHF/vHFML/AliAnalysisTaskSEHFResonanceBuilder.cxx

```diff
@@ -1531,7 +1531,7 @@ int AliAnalysisTaskSEHFResonanceBuilder::MatchResoToMC(AliAODMCParticle *partD,
         return 0;
 
     double momSumDaughters[3] = {partD->Px()+partLight->Px(), partD->Py()+partLight->Py(), partD->Pz()+partLight->Pz()};
-    for (auto iMother{commonMothers.size()-1}; iMother>=0; ++iMother) {
+    for (int iMother{commonMothers.size()-1}; iMother>=0; iMother--) {
```

Comment:
clang on mac complains at this line about a narrowing cast from unsigned long to int, for which compilation breaks here (c++11 narrowing)

Reply:
Thanks for pointing this out @mfasDa, [this PR](https://github.com/alisw/AliPhysics/pull/22829) should fix it.